cantools/database/can/formats/arxml.py

```diff
@@ -466,38 +329,35 @@ def load_system_signal(self, system_signal, decimal):
         offset = 0
         choices = None
 
-        compu_method_ref = system_signal.find(COMPU_METHOD_REF_XPATH,
-                                              NAMESPACES)
+        compu_method = self.find_unique_arxml_child(system_signal,
+                                                   [ "&PHYSICAL-PROPS",
+                                                     "SW-DATA-DEF-PROPS-VARIANTS",
+                                                     "&SW-DATA-DEF-PROPS-CONDITIONAL",
+                                                     "&COMPU-METHOD" ])
 
-        if compu_method_ref is not None:
-            compu_method = self.get_compu_method(compu_method_ref.text)
-            category = compu_method.find(CATEGORY_XPATH, NAMESPACES)
+
+        if compu_method is not None:
+            category = self.find_unique_arxml_child(compu_method, "CATEGORY")
 
             if category is None:
                 raise ValueError(
-                    'CATEGORY in {} does not exist.'.format(
-                        compu_method_ref.text))
+                    'CATEGORY in compu method {} does not exist.'.format(
+                        compu_method.find("SHORT-NAME").text))
 
             category = category.text
 
             if category == 'TEXTTABLE':
-                minimum, maximum, choices = self.load_texttable(
-                    compu_method,
-                    decimal)
+                minimum, maximum, choices = self.load_texttable(compu_method, decimal)
```

Comment:
Try not to make styling changes. Both makes the diff bigger and the styling is there for a reason. In this case to make lines shorter than about 85 character. This particular line is 86, which would often be acceptable, but let's not change it now.